### PR TITLE
Black Lives Matter announcement

### PR DESCRIPTION
**Pull request by grantr**
This work was done by @markusthoemmes, but since he's out I'm creating the PR. The text is the same as used in https://github.com/kubernetes/website/pull/21505 and https://github.com/kubernetes-sigs/kubebuilder/pull/1553.

/cc @evankanderson @mattmoor @tcnghia

### CODE REVIEW

File: layouts/partials/hooks/head-end.html [hunk 1]
```diff
@@ -2,3 +2,4 @@
 <!-- Webmaster Central ownership verification using "HTML tag" method -->
 <!-- https://www.google.com/webmasters/verification/details?hl=en&siteUrl=http://knative.dev/ -->
 <script src="/js/cookie-consent.js"></script>
+<script src="/js/script.js"></script>
```
Comment (mattmoor) [inline layouts/partials/hooks/head-end.html:5]: Add trailing newline:
```suggestion
<script src="/js/script.js"></script>

```

File: layouts/partials/navbar.html [hunk 2]
```diff
@@ -41,3 +41,10 @@
   </div>
   <div class="navbar-nav d-none d-xl-block">{{ partial "search-input.html" . }}</div>
 </nav>
+
+<section lang="en" id="announcement">
+  <h4>Black lives matter.</h4>
+  <p>We stand in solidarity with the Black community.<br>
+    <strong>Racism is unacceptable.</strong><br>
+    It conflicts with the <a href=https://github.com/knative/community/blob/master/VALUES.md>core values of the Knative project</a> and our community does not tolerate it.</p>
+</section>
```
Comment (mattmoor) [inline layouts/partials/navbar.html:50]: Add trailing newline:
```suggestion
</section>

```

File: static/js/script.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+        if (window.pageYOffset > 50) {
+            html.addClass("flip-nav");
+        } else {
+            html.removeClass("flip-nav");
+        }
+    }
+
+    window.addEventListener('resize', flipNav);
+    window.addEventListener('scroll', flipNav);
+});
```
Comment (mattmoor) [inline static/js/script.js:14]: Add trailing newline:
```suggestion
});

```

**Comment by mattmoor** [comment]
Thanks for picking this up.  The bot found a couple trailing newline nits and the CLA needs to be fixed, but otherwise this LGTM.

**Comment by grantr** [comment]
Since I can't push corrections to Markus' branch, replacing this with #202.

**Pull request closed without merging**